lib/src/utils.dart

```diff
@@ -44,3 +45,30 @@ String? deviceToDataName(Device device) => switch (device) {
   Device.SCIENCE => ScienceData().messageName,
   _ => null,
 };
+
+/// Helpful extension methods to convert GpsCoordinates into meters
+extension GpsCoordinatesToMeters on GpsCoordinates {
+  /// Number of meters per degree of latitude
+  static const metersPerLatitude = 111.32 * 1000;
+
+  /// Converts GpsCoordinates into meters
+  (num longitudeMeters, num latitudeMeters) get inMeters {
+    final metersPerLongitude = 40075 * cos(latitude * (pi / 180)) / 360 * 1000;
+    return (latitude * metersPerLatitude, longitude * metersPerLongitude);
+  }
+}
+
+/// Extension to convert coordinates in meters into Gps Coordinates
+extension MeterRecordToGps on (num, num) {
+  /// Convert a record of 2 doubles (which represent longitude and latitude coordinates) into Gps Coordinates
+  GpsCoordinates get toGpsCoordinates {
+    final latitude = $1 / GpsCoordinatesToMeters.metersPerLatitude;
+    final metersPerLongitude = 40075 * cos(latitude * (pi / 180)) / 360 * 1000;
+    return GpsCoordinates(latitude: latitude, longitude: $2 / metersPerLongitude);
+  }
+
+  /// Convert a record of 2 doubles (which represent longitude and latitude coordinates) into Gps Coordinates
+  /// 
+  /// Alias for [MeterRecordToGps.toGpsCoordinates]
+  GpsCoordinates toGps() => toGpsCoordinates;
```

Comment:
Let's choose one

Reply:
I added `toGps()` since autonomy has its own implementation and I didn't want to break it, we can remove it but we would have to modify autonomy

I'm thinking maybe just have a getter method called `toGps`

Reply:
Based on [this section](https://dart.dev/effective-dart/design#prefer-naming-a-method-to___-if-it-copies-the-objects-state-to-a-new-object) in Effective Dart, let's stick with `toGps()`